controllers/comments.js

```diff
@@ -0,0 +1,16 @@
+const { fetchCommentsByArticleId } = require('../models/comments');
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const articleId = req.params.article_id;
+  if (isNaN(articleId)) {
+    return res.status(400).json({ message: 'Invalid article ID' });
+  }
+  fetchCommentsByArticleId(articleId)
+    .then((comment) => {
+      if (comment.length === 0) {
+        return res.status(404).send({ message: 'Article not found'})
+      }
+      return res.status(200).json(comment);
```

Comment:
We should be returning an object with a key of comments here that contains the array of comments as the value